lib/features/rules/BpmnRules.js

```diff
@@ -896,3 +905,29 @@ function canCopy(collection, element) {
 
   return true;
 }
+
+function isOutgoingEventBasedGatewayConnection(connection) {
+
+  if (
+    connection &&
+    connection.source
+  ) {
+    return is(connection.source, 'bpmn:EventBasedGateway');
+  }
+}
+
+function areOutgoingEventBasedGatewayConnections(connections) {
+
+  connections = connections || [];
+
+  var result = false;
+
+  for (var i = 0; i < connections.length; i++) {
+    if (isOutgoingEventBasedGatewayConnection(connections[i])) {
+      result = true;
+      break;
+    }
+  }
+
+  return result;
```

Comment:
923-932 could be replaced simply with `connections.some(isOutgoingEventBasedGatewayConnection)`. It's implemented in IE11.